src/main/java/Alex.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
```

Comment:
I like how you chose explicitly which package from `java.util` rather than using a generic import.